specification/cognitiveservices/data-plane/InkRecognizer/readme.go.md

```diff
@@ -0,0 +1,33 @@
+## Go
+
+These settings apply only when `--go` is specified on the command line.
+
+```yaml $(go)
+go:
+  license-header: MICROSOFT_APACHE_NO_VERSION
+  namespace: inkrecognizer
+  clear-output-folder: true
+
+  directive:
+  # Replace the default client constructor with the one that enables TLS renegotiation.
```

Comment:
Also can you please confirm that your service requires this functionality and it's not just a copy/paste artifact?

Reply:
Removing. Confirmed that it's not needed.